Monika After Story/game/script-stories.rpy

```diff
@@ -0,0 +1,196 @@
+# Module for Monika story telling
+#
+# Stories will get unlocked one at by session
+# The unlocking logic has been added to script-ch30
+# The topic that triggers the Story menu is monika_short_stories
+# Topic is unlocked at the beginning of the game and is not
+# random
+
+
+# dict of tples containing the stories event data
+default persistent.story_database = dict()
+
+
+# store containing stories-related things
+init -1 python in mas_stories:
+
+    # pane constants
+    STORY_X = 680
+    STORY_Y = 40
+    STORY_W = 450
+    STORY_H = 640
+    STORY_XALIGN = -0.05
+    STORY_AREA = (STORY_X, STORY_Y, STORY_W, STORY_H)
+    STORY_RETURN = "I changed my mind"
+    story_database = dict()
+
+
+# entry point for stories flow
+label mas_stories_start:
+    show monika at t21
+    python:
+        import store.mas_stories as mas_stories
+
+        # build menu list
+        stories_menu_items = [
+            (mas_stories.story_database[k].prompt, k, not seen_event(k), False)
+            for k in mas_stories.story_database
+            if mas_stories.story_database[k].unlocked
+        ]
+
+        # also sort this list
+        stories_menu_items.sort()
+
+        # final quit item
+        final_item = (mas_stories.STORY_RETURN, False, False, False, 20)
+
+    # call scrollable pane
+    call screen mas_gen_scrollable_menu(stories_menu_items, mas_stories.STORY_AREA, mas_stories.STORY_XALIGN, final_item=final_item)
+
+    # return value? then push
+    if _return:
+        $ pushEvent(_return)
+
+    show monika at t11
+    return _return
+
+# Stories start here
+init 5 python:
+    addEvent(Event(persistent.story_database,eventlabel="story_tyrant",
+        prompt="The cat and the cock",unlocked=True),eventdb=store.mas_stories.story_database)
+
+label story_tyrant:
+    m 4b "Alright let's start the story."
+    m 3dfc "Ahem."
+    m 1i "A Cat caught a Cock and thought about reasonable excuses for eating him."
+    m "He accused him of being a nuisance by crowing at night; not letting men sleep."
+    m 1d "The Cock defended his action by saying this was for the benefit of men, as it wakes them for labor."
+    m 1tfb "The Cat replied, “you abound in apologies, but it’s time for breakfast.”"
+    m 1l "At that he made a meal of the Cock."
+    m 4d "The moral of this story is that: 'Tyrants need no excuse'."
+    m 1k "I hope you enjoyed this little story, [player]~"
+    return
+
+init 5 python:
+    addEvent(Event(persistent.story_database,eventlabel="story_despise",
+        prompt="The Fox",unlocked=False),eventdb=store.mas_stories.story_database)
+
+label story_despise:
+    m 3k "Ready for story time?"

```

Comment:
The "ready for story time?" and "Ahem" should be separated in a differnet label that can randomize the starts, especially since you use them in pratically every story here.

You could use the `MASQuipList` or just do something similar to the `limit_quips` in `random_limit_reached`. 

And then at the starts of these stories, call the shared label.

Reply:
Actually this was some sort of idea I had, to make Monika have a mannerism. I like the idea of not having it repeated on every topic, but I would like to see what writers think about her having said mannerism, probably should have added that to the reviewing part of the description of the pull request. If they agree on adding that mannerism I'll do the change.